arcaflow_plugin_pcp/pcp_plugin.py

```diff
@@ -91,6 +91,17 @@ def start_pcp(
             if "error" in pmlogconf_return[0]:
                 return pmlogconf_return
 
+        # Import the pmrep configuration file, if provided
+        if params.pmrep_conf:
+            pmrep_conf_path = "pmrep.conf"
+            print("Using provided pmrep configuration file")
+            f = open(pmrep_conf_path, "w")
+            f.write(params.pmrep_conf)
+            f.close
```

Comment:
Using a context manager for the write would give you the benefit of automatic cleanup on failure. In fact, with `pathlib`, the best way to do this is simply:

```
from pathlib import Path
...
    Path(pmrep_conf_path).write_text(params.pmrep_conf)
```

The open and close are implicit.